ogcore/output_plots.py

```diff
@@ -20,7 +20,7 @@ def plot_aggregates(
     reform_params=None,
     var_list=["Y", "C", "K", "L"],
     plot_type="pct_diff",
-    stationarized=True,
+    stationarized=False,
```

Comment:
@jdebacker I think we want `stationarized=True` in our default output plots.

Reply:
@rickecon Why wouldn't we want to use the non-stationarized series?

Reply:
I see -- the short run population dynamic introduce some wiggles that are not always helpful to view.